### PR TITLE
BN- Javadocs

### PR DESCRIPTION
**Pull request by rnyPlanet**

### CODE REVIEW

File: src/main/java/com/bobocode/svydovets/bibernate/exception/ConfigurationException.java [hunk 1]
```diff
@@ -7,6 +7,5 @@ public class ConfigurationException extends BibernateException {
 
     public ConfigurationException(String message, Throwable cause) {
         super(message, cause);
-        log.error(message);
```
Comment (rnyPlanet) [inline src/main/java/com/bobocode/svydovets/bibernate/exception/ConfigurationException.java:10]: I don't think it's necessary

File: src/main/java/com/bobocode/svydovets/bibernate/exception/EntityNotFoundException.java [hunk 2]
```diff
@@ -6,6 +6,5 @@
 public class EntityNotFoundException extends BibernateException {
     public EntityNotFoundException(String message) {
         super(message);
-        log.error(message);
```
Comment (rnyPlanet) [inline src/main/java/com/bobocode/svydovets/bibernate/exception/EntityNotFoundException.java:9]: I don't think it's necessary

File: src/main/java/com/bobocode/svydovets/bibernate/exception/EntityStateValidationException.java [hunk 3]
```diff
@@ -6,6 +6,5 @@
 public class EntityStateValidationException extends BibernateException {
     public EntityStateValidationException(String message) {
         super(message);
-        log.error(message);
```
Comment (rnyPlanet) [inline src/main/java/com/bobocode/svydovets/bibernate/exception/EntityStateValidationException.java:9]: I don't think it's necessary

File: src/main/java/com/bobocode/svydovets/bibernate/exception/EntityValidationException.java [hunk 4]
```diff
@@ -6,6 +6,5 @@
 public class EntityValidationException extends BibernateException {
     public EntityValidationException(String message) {
         super(message);
-        log.error(message);
```
Comment (rnyPlanet) [inline src/main/java/com/bobocode/svydovets/bibernate/exception/EntityValidationException.java:9]: I don't think it's necessary

**Review by rovein — APPROVED** [review]

**Review by ChonkaWork — APPROVED** [review]

**Pull request merged**